tests/testthat/test-shuffle_ard.R

```diff
@@ -66,32 +66,10 @@ test_that("shuffle_ard handles protected names", {
 
 test_that("shuffle_ard notifies user about warnings/errors before dropping", {
   expect_snapshot(
-    ard_ttest(
-      data = ADSL,
-      by = "ARM",
-      variable = "AGEGR1"
+    ard_continuous(
+      ADSL,
+      variables = AGEGR1
     ) |>
-      shuffle_ard() |>
-      as.data.frame()
+      shuffle_ard()
   )
 })
-
-test_that("shuffle_ard fills missing group levels if the group is meaningful", {
-  adsl_sub <- ADSL |> dplyr::filter(ARM %in% unique(ARM)[1:2])
-
```

Comment:
What do you think about adding something like this just to anticipate how it behaves when cardx is involved?

```
bind_ard(
  ard_continuous(ADSL, by = "ARM", variables = "AGE"),
  dplyr::tibble(group1 = "ARM", variable = "AGE", stat_name = "p", stat_label = "p", statistic = list(0.05))
) |>
  shuffle_ard() 
```

Reply:
Ah, great point! Added!
![image](https://github.com/insightsengineering/cards/assets/26774684/c4d1e25e-e2cb-48a4-ac0d-072b7ed2e8bd)